portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtension.testcase

```diff
@@ -96,6 +96,38 @@ definition {
 		ClientExtensionGeneral.viewEmptyRemoteTable();
 	}
 
+	@description = "Add Client Extension with DB Partitioning enabled."
+	@priority = "5"
+	test CanBeDisplayedWithDBPartitioning {
+		property database.partition.enabled = "true";
+		property database.types = "mysql";
+
+		task ("Given Client Extension admin page") {
+			ClientExtensionGeneral.goToRemoteAppsPortlet();
+		}
+
+		task ("When Click Add Button And When") {
```

Comment:
Hi @alexmds2004 could you fix the title task for `task ("When Click Add Button")`

Reply:
certo, ja fiz a modificacao